tests/python/frontend/tensorflow/test_forward.py

```diff
@@ -38,6 +38,8 @@
 from tvm import te
 from tvm import relay
 import tvm.relay.testing.tf as tf_testing
+from tensorflow.python.framework import dtypes
```

Comment:
We don't need to import this, just use `tf.float32` and `tf.float64` instead.

Reply:
removed